.github/workflows/automerge.yaml

```diff
@@ -43,9 +43,67 @@ jobs:
         run: |
           echo "Waiting for ~ 10 mins, PRT to be initiated." && sleep 600
 
+      - name: is PRT check Passed ?
+        run: |
+          api_endpoint="https://api.github.com/repos/${{github.repository}}/statuses/${{ github.head_ref }}"
+
+          function get_status() {
+            curl -s "$api_endpoint"  | jq -r '.[0].state'
+          }
+
+          function get_context() {
+            curl -s "$api_endpoint" | jq -r '.[0].context'
+          }
+
+          statuses_length=$(curl -s $api_endpoint | jq 'length')
+          if [ $statuses_length -eq 0 ]; then
+              echo "PRT failed to start ! Stopping."
+              exit 1
+          fi
+
+          status=$(get_status)
+          context=$(get_context)
+
+          if [ "$context" != "Robottelo-Runner" ]; then
+            echo "::error Failed to get Robottelo-Runner status"
+            exit 1
+          fi
+
+          counter=0
+          while [ "$status" != "success" ] && [ "$status" != "failure" ]; do
+            if [ $counter -gt 10 ]; then
+              echo "PRT failed/timeout"
+              exit 1
+            fi
+            echo "Waiting for PRT to complete....."
+            sleep 300
+            status=$(get_status)
+            echo "Robottelo-Runner : $status"
+            counter=$((counter+1))
+          done
+
+          if [ "$status" == "success" ]; then
+            echo "PRT Passed Successfully!"
+          else
+            echo "Robottelo-Runner : $status"
+            echo "::error PRT failed"
+            exit 1
+          fi
+
+      - name: Wait for other status checks to Pass
+        id: waitforstatuschecks
+        uses: lewagon/wait-on-check-action@v1.2.0
+        with:
+          ref: ${{ github.head_ref }}
+          repo-token: ${{ secrets.CHERRYPICK_PAT }}
+          wait-interval: 60
+          running-workflow-name: 'Wait for other status checks to Pass'
+          allowed-conclusions: success,skipped
+
       - id: automerge
         name: Auto merge of cherry-picked PRs.
         uses: "pascalgn/automerge-action@v0.15.5"
+        if: steps.waitforstatuschecks.outputs.status == 'success'
```

Comment:
You should also be able to echo the PRT check output using `set-output` in shell script. so that you check both of the checks non-PRT and PRT.

Reply:
Although the step is not necessary, as the lewagon/wait-on-check-action@v1.2.01 is responsible for failing the step, the PRT also exits when it fails. There is no need to collect the output result as the program exits at that point.

Reply:
But what if checks passed and PRT did not :) This step will still run.

Reply:
`exit 1`  will terminate the program with failure. There will not steps will be run.

Reply:
@jyejare this is the latest test result https://github.com/omkarkhatavkar/robottelo/actions/runs/3929759805/jobs/6718953226